### PR TITLE
auto-dismiss

### PR DESCRIPTION
**Pull request by brianteeman**
This allows us to have auto-dismiss with and without the manual dismiss x

Thanks @C-Lodder

### CODE REVIEW

File: src/js/alert/alert.js [hunk 1]
```diff
@@ -1,40 +1,39 @@
...
 
+    get autodismiss() { return this.getAttribute('auto-dismiss'); }
+
     get acknowledge() { return this.getAttribute('acknowledge'); }
 
     get href() { return this.getAttribute('href'); }
 
     /* Lifecycle, element appended to the DOM */
     connectedCallback() {
+      this.setAttribute('role', 'alertdialog');
```
Comment (wilsonge) [inline src/js/alert/alert.js]: Are we deliberately setting this back to hardcoded `role="alertdialog"`? (i.e. reverting https://github.com/joomla-projects/custom-elements/pull/119)

Comment (brianteeman) [reply src/js/alert/alert.js]: my mistake - should not have changesd

Comment (brianteeman) [reply src/js/alert/alert.js]: I just copy/pasted @c-lodder code after testing it for autodismiss. Didnt spot these other changes. I guess he edited an older version of the file. I will put them back and retest before updating this PR

Comment (brianteeman) [reply src/js/alert/alert.js]: OK should be correct now. I revert the dialog changes and retested and all seems good.

**Comment by brianteeman** [comment]
I "think" this might fix the error we are getting on the main repo as well

`10:11:18.724 SEVERE - http://localhost/test-install/media/vendor/joomla-custom-elements/js/joomla-alert.min.js?00d88ab1111327528710d8641fba25d9 0:1742 Uncaught TypeError: Cannot read property 'removeChild' of null`

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**